components/style/values/generics/font.rs

```diff
@@ -228,7 +228,7 @@ impl Default for KeywordSize {
 /// https://drafts.csswg.org/css-fonts-4/#font-style-prop
 #[allow(missing_docs)]
 #[cfg_attr(feature = "servo", derive(Deserialize, Serialize))]
-#[derive(Animate, Clone, ComputeSquaredDistance, Copy, Debug, MallocSizeOf,
+#[derive(Animate, Clone, ComputeSquaredDistance, Copy, Debug, MallocSizeOf, Hash,
```

Comment:
nit: tidy won't be happy about this.

Reply:
Tidy doesn't care (I have the pre-commit hook setup), I think its line limit is longer than this line. Happy to wrap it to the next line though.

Reply:
Oh, I meant because of the derive alphabetic sorting... Looks like a bug in tidy? Oh well :)

Reply:
Aha! Well, fixed now anyway.